### PR TITLE
Visual Restyle Update

### PR DESCRIPTION
**Pull request by dnenov**
### Purpose

This is a follow-up PR to the [Visual Restyle](https://github.com/DynamoDS/Dynamo/pull/13075) with a few minor tweaks, updates, and improvements.

- high-density design applied to various dialog windows
- various dialog windows can now allow resizing to accommodate for long text displays
- placement of buttons on the default Dynamo Message Box changed
- added **TitleBarButton** icons for dark and light theme
- light/dark theme for textboxes
- quality of life improvement - minimum grip height to scroll bars

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

- high-density design applied to various dialog windows
- various dialog windows can now allow resizing to accommodate for long text displays
- placement of buttons on the default Dynamo Message Box changed
- added **TitleBarButton** icons for dark and light theme
- light/dark theme for textboxes
- quality of life improvement - minimum grip height to scroll bars

### Reviewers

@reddyashish 

### FYIs

@Amoursol

### CODE REVIEW

**Comment by Amoursol** [comment]
@dnenov would it be possible to please edit the PR submission and add in a description of what was improved? It helps with future referral when we look back :)

File: src/DynamoCoreWpf/DynamoCoreWpf.csproj [hunk 1]
```diff
@@ -811,6 +827,22 @@
     <Resource Include="UI\Images\NodeStates\frozen-64px.png" />
     <Resource Include="UI\Images\NodeStates\hidden-64px.png" />
     <Resource Include="UI\Images\NodeStates\info-64px.png" />
+    <Resource Include="UI\Images\TitleBarButtons\close-darktheme-default-16px.png" />
```
Comment (reddyashish) [inline src/DynamoCoreWpf/DynamoCoreWpf.csproj:830]: These are already added above. Duplicated.

**Comment by reddyashish** [comment]
Thanks @dnenov, Also can we delete the old images if nothing is using it.

File: src/DynamoCoreWpf/DynamoCoreWpf.csproj (hunk 1 above)
Comment (dnenov) [reply src/DynamoCoreWpf/DynamoCoreWpf.csproj:830]: Can we go over those in chat, @reddyashish ?

**Review by reddyashish — APPROVED** [review]

**Pull request merged**